pkgs/by-name/sh/shellstorm-qt/package.nix

```
@@ -0,0 +1,42 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  qt6,
+}:
+
+stdenv.mkDerivation {
+  pname = "shellstorm-qt";
+  version = "1.0";
```

```suggestion
  version = "0-unstable-YYYY-MM-DD";
```
where that is the date of the last commit.